### DISCUSSION TITLE
Extendable config

### DISCUSSION THREAD

**Discussion by marcalexiei** [rome/tools#4036] [Suggestions]
I just finished to read the documentation and I found out that, at the moment, there is no mention of a way top extend an existing configuration (basically an `extends` key).

Searching among the issues I found #1596.
In particularly in this comment https://github.com/rome/tools/issues/1596#issuecomment-1324825243 is stated:

> Since the rewrite to Rust, we haven't been able to port ALL the old features we had. This is one of them, we plan to add it again soon. Unfortunately, I can't give you a timeline. Once there, the documentation will be updated!

Since this issue is closed, it might be useful to create a new one to keep track of this feature?

**Comment by ematipico** [comment]
We (the staff team) usually open issues once we know a feature is being developed, to avoid to overwhelm ourselves with issues. This help us to better track the works and to avoid losing some issues that fell behind. And avoid scope creep.

Each month we try to narrow down which features we want to develop based on the team's priority, how the community is reacting, what we deem important for the community, and other factors. Sometime we have to make hard decisions.

You're free to open an issue if you want, but it could take some time before it will be picked up by one of us. I just hope we don't lose it! It happened before... 😔

**Reply by marcalexiei** [reply]
> You're free to open an issue if you want, but it could take some time before it will be picked up by one of us. I just hope we don't lose it! It happened before... 😔

No problem at all, I created #4038. 
Thanks!
